opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -861,9 +933,7 @@ namespace Opm {
         Dune::Timer perfTimer;
         perfTimer.start();
 
-        if ( ! wellsActive() ) {
```

Comment:
Is it wrong to leave this in place? If there are no wells, then the network need not be updated either?

Reply:
From the  result in the commercial simulator, it looks like the network pressures are updated before the wells are opening.  Doing the same just make the comparison plot look better.